packages/workflows/src/doc/IParamsCreateWorkflow.ts

```diff
@@ -0,0 +1,107 @@
+import { AbstractSession } from "@zowe/imperative";
+import { accessT } from "./ICreateWorkflow";
```

Comment:
Maybe I got this one wrong, but since `IParamsCreateWrofklow` and IParamsCreateWorkflowLocal` only differ in `keepFiles` and `customDir`, is it worth trying to reduce the duplication by extending `IParamsCreateWorkflow`  (Or creating a common interface) ?